### PR TITLE
landing analytics v2

### PR DESCRIPTION
**Pull request by wdwatkins**

### CODE REVIEW

**Comment by mwernimont** [comment]
Hmm...you don't usually put a script tag inside of a header, you either have them in the head of the page, or just before the body tag finishes.  I don't think this is where we would want this script tag.

**Comment by wdwatkins** [comment]
@mwernimont in the vizzies or tags mustache then?

**Comment by wdwatkins** [comment]
Doesn't seem like there is a general page template in the `inst/landing` stuff.  Maybe it is using the default vizlab template?

**Comment by mwernimont** [comment]
In inst/templates/fullPage.mustache looks to be the best bet, theres already a GA analytics script in there too.

**Comment by wdwatkins** [comment]
Ok, yeah we should probably update the existing GA code and just make sure the landing page function utilizes that part of the template.

**Comment by wdwatkins** [comment]
Ok @mwernimont, should be better now

**Comment by wdwatkins** [comment]
This will add the USGS analytics to all pages, so partially fulfills #137.  I will leave the rest of that issue to a future PR, which involve modifying all the templates that have GA baked in.

File: R/publish.R [hunk 1]
```diff
@@ -322,7 +322,8 @@ publish.landing <- function(viz){
   pageviz$context <- list(sections = c("owiNav", "header", names(viz_info)), #names of section ids
                           resources = c("landingCSS", "owiCSS", "jquery", "appJS"),
                           header = "usgsHeader",
-                          footer = "usgsFooter")
+                          footer = "usgsFooter", 
+                          info = list(`analytics-id` = "UA-78530187-11"))
```
Comment (wdwatkins) [inline R/publish.R]: Not sure if we want to hard code this in or not?  In theory we wouldn't ever change this, but it could be passed in.

**Pull request merged**